### PR TITLE
Add pretty option to serializer

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

**Comment by torkleyy** [comment]
That's what the serialized stuff looks like:

```rust
(
    boolean: false,
    float: 2.180000066757202,
    map: {
        1: '2',
        3: '5',
        8: '1',
        0: '1',
    },
    nested: (
        a: "Hello from \"RON\"",
        b: 'b',
    ),
)
```

File: src/ser/mod.rs [hunk 1]
```diff
@@ -207,13 +241,15 @@ impl<'a> ser::Serializer for &'a mut Serializer {
         self.output += variant;
         self.output += "(";
         value.serialize(&mut *self)?;
-        self.output += ",)";
+        self.output += ")";
```
Comment (torkleyy) [inline src/ser/mod.rs]: This causes the compilation issue. Any idea why?

Comment (torkleyy) [inline src/ser/mod.rs]: Never mind, found the issue.

**Comment by kvark** [comment]
:shipit:
bors r+

**Comment by kvark** [comment]
cmon buddy!
bors r+

**Pull request merged**